src/content/docs/getting-started/intro.md

```diff
@@ -4,55 +4,15 @@ description: An overview of Membrane and its uses
 ---
 
 :::note
-These docs are still a work-in-progress. If you have any questions, please [reach out to us on discord](/discord/)
+These docs are open source. Feel free to open a PR [on GitHub](https://github.com/membrane-io/docs) for fixes/improvements. If you have any questions, please [reach out to us on discord](/discord/).
```

Comment:
Even though the docs _are_ still a work in progress after this PR, they'll be usable. With this new messaging the hope is to get people reading and even improving them.